distros/ronin/homescreens.json

```diff
@@ -1,15 +1,12 @@
 {
   "homescreens": [
     [
-      ["apps", "search"],
```

Comment:
Not sure what is the effect of this removal? (it's the only non-obvious one, so you may want to explain it in the commit message)